src/PrismicNextLink.tsx

```diff
@@ -51,5 +51,14 @@ export const PrismicNextLink = React.forwardRef<
 		rel = restProps.rel;
 	}
 
-	return <Link ref={ref} {...attrs} {...restProps} href={href} rel={rel} />;
+	let text: string | undefined;
+	if (field && "text" in field) {
+		text = field.text;
+	}
+
+	return (
+		<Link ref={ref} {...attrs} {...restProps} href={href} rel={rel}>
+			{children || text}
```

Comment:
See https://github.com/prismicio/prismic-react/pull/205#discussion_r1735435335

Reply:
Replied there 🙂

Reply:
Once we resolve the conversation there, I will approve this PR. 🙂

Reply:
Done 👍